Cannot find module '@directus/shared/types'

### Describe the Bug

Hi, I'm currently facing an issue starting up Directus on my server. 
It says: `Cannot find module '@directus/shared/types'`, but this is not a dependency I use.
I'm running node v16.13.1 and npm v8.1.2, which should meet the requirements of Directus. 
I've even tried `npm i --save @directus/shared`, but this, unfortunately, did not work. 
Does anyone know why this is happening?

### To Reproduce

Not sure how to reproduce

### Errors Shown

```
Cannot find module '@directus/shared/types'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! project-name@1.0.0 start: `npx directus start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the project-name@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-11-24T10_46_19_940Z-debug.log
```

### What version of Directus are you using?

9.21.0

### What version of Node.js are you using?

v16.13.1

### What database are you using?

PostgreSQL 12

### What browser are you using?

Not related, i'm using server os: Debian GNU/Linux 10

### How are you deploying Directus?

Running locally with pm2

--- a/packages/shared/package.json
+++ b/packages/shared/package.json
@@ -50,6 +50,7 @@
 	},
 	"files": [
 		"dist",
+		"*.d.ts",
 		"!**/*.test.js",
 		"!**/*.d.ts.map"
 	],

